controllers/leaderelection/coordination_api/http_client.go

```diff
@@ -74,7 +73,7 @@ func (c *podCoordinationHttpClient) PollPod(ctx context.Context, pod coreV1.Pod)
 			pod.GetObjectMeta().GetName(), resp.Status)
 	}
 
-	body, err := ioutil.ReadAll(resp.Body)
+	body, err := io.ReadAll(resp.Body)
```

Comment:
Was this change pushed by accident?

Reply:
In Go 1.19, the io/ioutil package has been deprecated, and the functionality has been moved to io package